POMPOM/POMPOM/UI/ClothPickerView/CategoryGrid.swift

```diff
@@ -31,19 +32,17 @@ struct CategoryGrid: View {
                                 offSet = Constant.screenWidth / 2 - 120
                             case .bottom:
                                 offSet = Constant.screenWidth / 2 - 190
-                            case .socks:
-                                offSet = Constant.screenWidth / 2 - 265
                             case .shoes:
+                                offSet = Constant.screenWidth / 2 - 265
```

Comment:
265 는 어떤의미의 값인가요 ?? 👀 
```suggestion
                                let margin: CGFloat = 265.0
                                offSet = Constant.screenWidth / 2 - margin
```

값에 대해 명명되면 어떤 값이고 빼야하는 이유에 이해할 수 있을 것 같습니다 !

Reply:
값에 이름을 지어주었습니다 ㅎㅎ😇